Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."MDVFragments_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3135 (class 0 OID 0)
+-- Dependencies: 208
+-- Name: MDVFragments_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."MDVFragments_id_seq" OWNED BY public."MDVFragments_from_excel".id;
+
+
+--
+-- TOC entry 207 (class 1259 OID 17743)
+-- Name: MFA model metabolites ; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."MFA model metabolites " (
+    id integer NOT NULL,
+    mapping_id text NOT NULL,
+    met_id text NOT NULL,
+    met_elements text[],
+    met_atompositions integer[],
+    met_symmetry_elements text[],
+    met_symmetry_atompositions integer[],
+    used boolean,
+    comment text,
+    met_mapping json,
+    base_met_ids text[],
+    base_met_elements json,
+    base_met_atompositions json,
+    base_met_symmetry_elements json,
+    base_met_symmetry_atompositions json,
+    base_met_indices integer[]
+);
+
+
+ALTER TABLE public."MFA model metabolites " OWNER TO postgres;
+
+--
+-- TOC entry 206 (class 1259 OID 17741)
+-- Name: MFA model metabolites _id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."MFA model metabolites _id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."MFA model metabolites _id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 3136 (class 0 OID 0)
+-- Dependencies: 206
+-- Name: MFA model metabolites _id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: postgres
+--
+
+ALTER SEQUENCE public."MFA model metabolites _id_seq" OWNED BY public."MFA model metabolites ".id;
+
+
+--
+-- TOC entry 204 (class 1259 OID 17729)
+-- Name: MFA model reactions table; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public."MFA model reactions table" (
+    id integer NOT NULL,
+    model_id text NOT NULL,
+    rxn_id text NOT NULL,
+    rxn_name text,
+    equation text,
+    subsystem text,
+    gpr text,
+    genes text[],
+    reactants_stoichiometry double precision[],
+    products_stoichiometry double precision[],
+    reactants_ids text[],
+    products_ids text[],
+    lower_bound double precision,
+    upper_bound double precision,
+    objective_coefficient double precision,
+    flux_units text,
+    fixed boolean,
+    free boolean,
+    reversibility boolean,
+    weight double precision,
+    used boolean,
+    comment text
+);
+
+
+ALTER TABLE public."MFA model reactions table" OWNER TO postgres;
+
+--
+-- TOC entry 205 (class 1259 OID 17732)
+-- Name: MFA model reactions table_id_seq; Type: SEQUENCE; Schema: public; Owner: postgres
+--
+
+CREATE SEQUENCE public."MFA model reactions table_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE public."MFA model reactions table_id_seq" OWNER TO postgres;
```

Comment:
Be sure to remove all white spaces and the `table` part from the sequences as well.

Reply:
done